### ISSUE TITLE
Issue debugging Wurst in Eclipse

### ISSUE THREAD

**Issue by Mimerme** [Wurst-Imperium/Wurst-Client#543]
After having followed the instructions in the wiki I get this error

Exception in thread "main" joptsimple.MissingRequiredOptionException: Missing required option(s) ['userProperties', 'accessToken', 'version']

I assume that something with the arguments when launching the application were incorrect or unspecified. But I'm having trouble assigning these arguments, tell me if I am right or wrong and how to fix this problem.

**Comment by Mimerme** [comment]
I feel as though I may have implemented the patch incorrectly. I kind of halfassed it. When I applied the patch at first in the root of the eclipse project /eclipse/Client it said that it could not find the directories under the label src/net/minecraft/etc... I went through the decompiled MCP folders and did not find anything with the correct folder structure. Is the patch wrong or am I looking in the wrong places?
